exporter/kafkaexporter/authentication.go

```diff
@@ -68,6 +84,29 @@ func configurePlaintext(config PlainTextConfig, saramaConfig *sarama.Config) {
 	saramaConfig.Net.SASL.Password = config.Password
 }
 
+func configureSCRAM(config SCRAMConfig, saramaConfig *sarama.Config) error {
+	saramaConfig.Net.SASL.Enable = true
+	saramaConfig.Net.SASL.User = config.Username
+	saramaConfig.Net.SASL.Password = config.Password
+	if config.Algorithm != "sha256" && config.Algorithm != "sha512" {
```

Comment:
I feel like a switch statement works better here.

Reply:
Yeah! I totally agree.
I'll change it.